docs/operator-guide/cd/deploy-rpm.md

```diff
@@ -0,0 +1,310 @@
+
+3. Configure AWX via UI:
+
+  a. Once AWX operator is deployed, navigate to its UI using the hostname you specified earlier:
+
+    ![AWX main menu](../../assets/operator-guide/rpm/awx-main-menu.png "AWX main menu")
+
+  b. (Optional) Configure OIDC. Navigate to **Settings** -> **Generic OIDC settings** and specify the required fields:
+
+  * **OIDC Key**: Enter **awx**;
+  * **OIDC Secret**: This is the client secret data taken from the secret you specified in the **eso.secretName** field of the **chart/clusters/\<cluster_name\>/custom/awx-operator/values.yaml** file;
+  * **URL**: Specify the Keycloak realm URL.
+
+    ![Generic OIDC settings](../../assets/operator-guide/rpm/awx-oidc-settings.png "Generic OIDC settings")
+
+  c. Navigate to **Resources** -> **Credentials** and create two credentials:
+
+    **For Git repository**: These credentials of the **Source Control** type must contain the SSH key you provided your Ansible Git repo with:
+
+    ![Git credentials](../../assets/operator-guide/rpm/awx-git-credentials.png "Git credentials")
+
+    **For AWS instances**: These credentials of the **Machine** type must contain the SSH key that grants access to your AWS EC2 instances:
+
+    ![AWS EC2 credentials](../../assets/operator-guide/rpm/awx-ec2-credentials.png "AWS EC2 credentials")
+
+  d. Navigate to **Resources** -> **Project** and create Ansible Project that with your Git repository you created at the beginning of the integration:
+
+    ![Create AWX project](../../assets/operator-guide/rpm/awx-create-project.png "Create AWX project")
+
+  e. Navigate to **Resources** -> **Inventories** and create an inventory using the **Add inventory** option:
+
+    ![AWX inventory](../../assets/operator-guide/rpm/awx-inventory.png "AWX inventory")
+
+  f. Navigate to **Resources** -> **Hosts** and add your hosts' IP addresses:
+
+    ![AWX hosts](../../assets/operator-guide/rpm/awx-hosts.png "AWX hosts")
+
+  g. Navigate to **Resources** -> **Templates**. Create job template called **package-install** as follows:
+
+    ![AWX template](../../assets/operator-guide/rpm/awx-template.png "AWX template")
+
+:::note
+  Don't forget to select the **Prompt on launch** checkbox for **inventory**, **limit**, and **variables** fields.
+:::
+
+4. Create application with the **rpm-package** deployment option. Refer to the [Add Application](../../user-guide/add-application.md) page for more details:
+
+  ![Create RPM application](../../assets/operator-guide/rpm/rpm.gif "Create RPM application")
+
+5. When [creating environments](../../user-guide/manage-environments.md#add-a-new-environment), in the **Deploy pipeline template** field, select **deploy-ansible-awx**:
+
+  ![Create Ansible environment](../../assets/operator-guide/rpm/ansible-deploy-ansible-awx.png "Create Ansible environment")
+
+6. When deploying application, enable the **Values override** option to apply configuration from the Git repository:
```

Comment:
6. (Optional) When deploying a component, you can enable the **Values override** option if you want an additional role, created in the Ansible Git repository, to be executed specifically for this component. This role should follow the naming convention **\<codebase-name\>-\<deployment-flow-name\>-\<environment-name\>**.